webui/src/pages/bug/Bug.tsx

```diff
@@ -88,9 +89,11 @@ function Bug({ bug }: Props) {
       <div className={classes.container}>
         <div className={classes.timeline}>
           <TimelineQuery id={bug.id} />
-          <div className={classes.commentForm}>
-            <CommentForm bugId={bug.id} />
-          </div>
+          <ReadonlyHidden>
```

Comment:
Does this hide the elements at runtime (i.e. they're still present in the dom?), or do they not get rendered at all?

I suppose - either way - this is safe because the mutating routes don't get mounted at all.

Reply:
This causes them to not get rendered at all - they're not present in the DOM.

But yes, as you point out, the mutating routes are inaccessible in any case.